json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -183,4 +185,32 @@ protected function get_plugin_updates( $plugin_file ) {
 		}
 		return null;
 	}
+
+	/**
+	 * Get custom action link tags that the plugin is using
+	 * Ref: https://codex.wordpress.org/Plugin_API/Filter_Reference/plugin_action_links_(plugin_file_name)
+	 * @return array of plugin action links (key: link name value: url)
+	 */
+	 protected function get_plugin_action_links( $plugin_file ) {
+		 $formatted_action_links = array();
+
+		 $action_links = apply_filters( 'plugin_action_links', null, $plugin_file, null, 'all' );
```

Comment:
Could you add a docblock here? Something like `/** This filter is documented in src/wp-admin/includes/class-wp-plugins-list-table.php */` should work and will make our parser happier.

Thanks!